Sources/Sentry/SentryDebugMeta.m

```diff
@@ -16,7 +16,8 @@ - (instancetype)init {
     [serializedData setValue:self.imageAddress forKey:@"image_addr"];
     [serializedData setValue:self.imageSize forKey:@"image_size"];
     [serializedData setValue:[self.name lastPathComponent] forKey:@"name"];
-
+    [serializedData setValue:self.imageVmAddress forKey:@"image_vmaddr"];
```

Comment:
Can we use literal syntax here for all of them? It makes the code easier to read IMO.
```suggestion
    serializedData[@"image_vmaddr"] = self.imageVmAddress;
```

Reply:
Using `setValue` is very important, since if the value is `nil` (which in that case can happen) it doesn't do anything.
vs. `serializedData[@"image_vmaddr"] = self.imageVmAddress;` would set `nil` and when we try to serialize this to JSON it would fail.

Reply:
If I'm not mistaken `mutableDictionary[@"someKey"] = value;` calls [`setObject:forKeyedSubscript`](https://developer.apple.com/documentation/foundation/nsmutabledictionary/1574187-setobject) and doesn't set nil, but removes the key from the dictionary.

Reply:
Ok, point taken 😅 
Anyway, we are using `setValue` everywhere else (in every other interface class) just changing this here may seem odd.